Shopify/cli
# Sync Flow Trigger SDL

### WHY are these changes introduced?

Fixes https://github.com/Shopify/flow/issues/19878

### WHAT is this pull request doing?
This PR adds functionality to sync SDL data for flow triggers.

### How to test your changes?
- Create partner trigger with schema patch
- Deploy app
- See that schema patch exists in task_return_fields table

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
